domains.json

```diff
@@ -68,6 +71,11 @@
     "1inch.io",
     "mantle.xyz",
     "friend.tech",
+    "balancer.fi",
```

Comment:
the main source of inclusion to the whitelist is by getting listed on defillama as a protocol, the whitelist here is only for protocols that arent listed on defillama but that are still important

balancer is listed on defillama with that url, so including it here again is not needed

Reply:
Interesting - any thoughts on why it wouldn't be showing up as green?

![image](https://github.com/DefiLlama/url-directory/assets/1376034/456c1a61-d401-4165-ae9b-14f8c58c1d63)